### PR TITLE
Fix invalid reference

### PR DESCRIPTION
**Pull request by qiaozha**
to fix issue #5154

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: lib/services/resourceManagement/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Srinivasan, Vivek <visriniv@microsoft.com>"
   ],
-  "version": "7.3.0",
+  "version": "8.0.0",
```
Comment (ramya-rao-a) [inline lib/services/resourceManagement/package.json:9]: Why would this change need a major version update?

Comment (qiaozha) [reply lib/services/resourceManagement/package.json:9]: Just in case some user may use this reference explicitly

Comment (qiaozha) [reply lib/services/resourceManagement/package.json:9]: Actually I think we should remove the dependency in ms-rest-azure https://github.com/Azure/azure-sdk-for-node/issues/5157 here

Comment (ramya-rao-a) [reply lib/services/resourceManagement/package.json:9]: Though moment is imported in these files, it is not exported and therefore a user would have no visibility to it. So, a major version update is not needed.

Comment (qiaozha) [reply lib/services/resourceManagement/package.json]: Okay, Changed. Please review it again.

**Pull request merged**